docs/cypress/e2e/patterns.cy.js

```diff
@@ -0,0 +1,51 @@
+const hostURL = Cypress.env('HOST_URL') || 'http://localhost:8000';
+const patternsURL = hostURL + '/patterns/overview/all-patterns/';
+const statusURL = hostURL + '/patterns/overview/status/'
+
+describe('Cypress Test of patterns page', () => {
+  const navLink = [];
+  before(() => {
+    cy.visit(patternsURL);
+    cy.leftnavTraverse(navLink);
+  });
+
+  it('check if all links of table of contents direct to the content', () => {
+    navLink.forEach((page) => {
+      cy.visit(page);
+      if (!page.includes('overview')) {
```

Comment:
We can also create a new array that contains all the pages except the `overview` page and then traverse that array instead of wrapping the test inside `if` block

Reply:
Done